src/integration/common/a11y.js

```diff
@@ -1,11 +1,16 @@
+const { within } = require('@testing-library/dom');
+
 const { amp, canonical } = global;
 
 export default ({ skipToContentText, headlineText }) => {
   [amp, canonical].forEach(page => {
     describe(`And using ${page.platform}`, () => {
       it('I can see a skip to content link that links to the main content of the page', () => {
         const skipToContentEl = page.getByText(skipToContentText);
-        const mainContentEl = page.getByText(headlineText);
+        const { getByText } = within(
+          page.document.querySelector('h1[id="content"]'),
```

Comment:
I don't think this selector needs to change because we have an assertion that explicitly checks for an id of content on line 16

Reply:
The MAP a11y tests fail because there are 2 elements with the same text in the HTML - the VisuallyHiddenText & StyledFauxHeadline. Unless we fix the underlying issue (which I believe you had raised here: https://github.com/bbc/simorgh/issues/5688 but it has since been closed), then this won't work.